generator.yaml

```diff
@@ -11,22 +11,22 @@ operations:
 resources:
   Bucket:
     fields:
-      # AccelerateConfiguration:
-      #   from:
-      #     operation: PutBucketAccelerateConfiguration
-      #     path: AccelerateConfiguration
+      AccelerateConfiguration:
```

Comment:
Would be cool if we named this just Accelerate to aling with the way that CORS, Logging, Encryption and Website fields work...